guardian/amigo
# feat: Allow the Data Science team to use AMIgo

## What does this change?
Expand the Google group check to the Data Science group, to enable them to use AMIgo.

An alternative would be to add colleagues into the `departmentGroupId` Google group...

## How to test
- Create the SSM Parameter `/CODE/deploy/amigo/auth.google.dataScientistsGroupId`
- Deploy this branch to CODE
- Check you can still access AMIgo CODE
- Check that colleagues in the data science group can access AMIgo CODE

## TODO before merge
- [ ] Create the SSM Parameter `/PROD/deploy/amigo/auth.google.dataScientistsGroupId`
